andela/ah-backend-iroquois
# feature(paginate articles): Pagination support for articles.

#### What does this PR do?
-  It paginates the articles created by User
#### Description of Task to be completed?
- Pagination support is lacking for our articles. Reading can be a nightmare when we have so many articles. This PR solves that problem.
#### How should this be manually tested?

- The developer should use the Get HTTP Method and specifies the page number.
This the URL you use.
` /api/articles/?page={page_number}`
Below is the sample of the URL, testing page 5.
`/api/articles/?page=5`

#### What are the relevant pivotal tracker stories?
[159952015](https://www.pivotaltracker.com/story/show/159952015)
